graphql_compiler/schema_transformation/rename_query.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2019-present Kensho Technologies, LLC.
+from copy import deepcopy
+
+from graphql.language import ast as ast_types
+from graphql.language.visitor import Visitor, visit
+
+from .utils import QueryStructureError
+
+
+def rename_query(ast, renamings):
+    """Translate all types and root fields (fields of query type) using renamings.
+
+    Most fields (fields of types other than the query type) will not be renamed. Any type
+    or field that does not appear in renamings will be unchanged.
+    Roughly speaking, rename_query renames the same elements as rename_schema.
```

Comment:
I think it'd be best that you just explained that after we rename a schema, we also need to rename the queries intended for that schema in this docstring block.

Reply:
The initially intended use case is kind of backwards -- rename schema S to S' so that S' can be merged without conflicts, then query Q' written by the user in S' gets renamed to Q and can actually be run against a database. This is somewhat hard to explain though. What if I just say "This function is intended to be used in conjunction with rename_schema"?

Reply:
How about: 
"The intended use of this function is to fix queries that were invalidated because of a schema renaming by analogously renaming the elements in the query. "